.ci.yaml

```diff
@@ -91,6 +91,10 @@ targets:
           "builder",
           "git"
         ]
+      ignore_cache_paths: >-
+        [
+          "builder/src/flutter/prebuilts/SDKs"
+        ]
```

Comment:
Should this be under `Mac builder_cache`?

Reply:
Oh right! Thank you! Good catch.